src/components/common/ColonyActions/ActionDetailsPage/DefaultMotion/MotionPhaseWidget/StakingWidget/StakingInput/StakingSlider.tsx

```diff
@@ -58,14 +61,19 @@ const StakingSlider = ({
         isLoading={isLoadingData}
         remainingToStake={remainingToStake}
         enoughTokensToStakeMinimum={enoughTokensToStakeMinimum}
+        enoughReputationToStakeMinimum={enoughReputationToStakeMinimum}
         limit={userStakeLimitDecimal}
         handleLimitExceeded={setLimitExceeded}
       />
-      {displayLabel && (
+      {displayErrMsg && (
         <StakingValidationMessage
+          enoughReputationToStakeMinimum={enoughReputationToStakeMinimum}
           enoughTokensToStakeMinimum={enoughTokensToStakeMinimum}
-          userActivatedTokens={userActivatedTokens}
           userMinStake={userMinStake}
+          userActivatedTokens={userActivatedTokens}
+          userMaxStake={userMaxStake}
+          remainingToStake={remainingToStake}
+          nativeTokenDecimals={nativeTokenDecimals}
```

Comment:
Again, without a significant redesign, it seems we are required to duplicate a lot of logic inside the Staking Slider so that it and the controls stay in sync.  (staking slider cannot access colony context nor staking widget provider because it's used inside the objection dialog).